### PR TITLE
Added reactions capabilities to T3

### PR DESCRIPTION
**Pull request by alongd**
Added an RMG runner script to execute RMG as a queue job o a server
Added a settings file
various other modifications
tests added

### CODE REVIEW

**Review by Laxzal — DISMISSED** [review]
Hi Alon, could you check out the comments before I approve the pull request

File: devtools/install_arc.sh [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
 cd ..
-# RMG-database
 git clone https://github.com/ReactionMechanismGenerator/RMG-database
-# RMG-Py
 git clone https://github.com/ReactionMechanismGenerator/RMG-Py
 cd RMG-Py
 export PYTHONPATH=$PYTHONPATH:$(pwd)
 echo "export PYTHONPATH=$PYTHONPATH:$(pwd)" >> ~/.bashrc
-# compile RMG
+mamba env update -n rmg_env -f environment.yml
```
[the remark this answers is not in the document]
Comment (alongd) [reply devtools/install_arc.sh:14]: Do you think installing mamba at the outset would be a good solution?

File: environment.yml [hunk 2]
```diff
@@ -7,56 +7,62 @@ channels:
...
+  - mako
+  - pytables
+  - pytorch::pytorch >=1.12.0
+  - anaconda::pytest
+  - conda-forge::torchani
+  - conda-forge::xtb
+  - mkdocs-material >=5.1.7
+  - mkdocs-material-extensions
+  - mock
+  - julia
```
[the remark this answers is not in the document]
Comment (alongd) [reply environment.yml:67]: I agree. It might be a leftover, since we removed the RMS adapters and we run RMG in a subprocess within its own env. I'll remove `julia` from T3's env

File: environment.yml [hunk 3]
```diff
@@ -7,56 +7,62 @@ channels:
...
+  - pyrms
```
[the remark this answers is not in the document]
Comment (alongd) [reply environment.yml:68]: Excellent, added

File: devtools/install_arc.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (Laxzal) [reply devtools/install_arc.sh:14]: For sure installing mamba from the get go is the best solution for quick installation of environments but I do think we need to take into account that some people won't have mamba installed. I used conda all my life until I came to our lab and changed to mamba. And even then, installing mamba the old fashioned way of:
1. install anaconda
2. install mamba over the anaconda base environment
Can be a real challenge as anaconda will solve for a very long time and find a lot of conflicts and may never proceed due to the conflicts. However, the user can install mambaforge which is an easy installation and quick.

So, in short, I think we should still continue to support a conda installation and have the sh file check if mamba is installed, and if so change to that as the installer. However, we should include in our installation documentation for RMG/T3/ARC that installing mambaforge is strongly recommended.

[the remark this answers is not in the document]
Comment (Laxzal) [reply devtools/install_arc.sh:14]: echo "Determining whether Mamba is installed..."
if [ $(which mamba) ]; then
echo "Mamba found!"
COMMAND_PKG=mamba
else
echo "Reverting to Conda..."
COMMAND_PKG=conda
fi

[the remark this answers is not in the document]
Comment (Laxzal) [reply devtools/install_arc.sh:14]: probably not the best way to determine if mamba is installed but it works

File: environment.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Laxzal) [reply environment.yml:67]: Maybe don't remove yet because as far as I can see when I attempt to run T3 locally, it won't switch env when running RMG files, so it needs julia still

**Review by Laxzal — APPROVED** [review]
